tests/Node/Expression/NameTest.php

```diff
@@ -34,7 +34,7 @@ public function getTests()
         $env = new Environment($this->createMock(LoaderInterface::class), ['strict_variables' => true]);
         $env1 = new Environment($this->createMock(LoaderInterface::class), ['strict_variables' => false]);
 
-        $output = '(isset($context["foo"]) || array_key_exists("foo", $context) ? $context["foo"] : (function () { throw new RuntimeError(\'Variable "foo" does not exist.\', 1, $this->source); })())';
+        $output = '(array_key_exists("foo", $context) ? $context["foo"] : throw new RuntimeError(\'Variable "foo" does not exist.\', 1, $this->source))';
```

Comment:
I think you must put a backslash before `array_key_exists` to benefit from the performance improvement of PHP 8.0.

Reply:
Actually, no, because these classes are not in a namespace.